swarm/storage/hasherstore.go

```diff
@@ -26,15 +26,23 @@ import (
 	"golang.org/x/crypto/sha3"
 )
 
+const (
+	NoOfStorageWorkers = 150
```

Comment:
This is not used anywhere else outside the package, so does not need to be exported. also, please rename to `maxStorageWorkers` or similar

Reply:
Done